### PR TITLE
Add MicroProfile 7.0 repeats to FaultTolerance buckets

### PR DESCRIPTION
**Pull request by tevans78**
This PR replaces https://github.com/OpenLiberty/open-liberty/pull/28746

Update MP Fault Tolerance buckets to repeat against MP 7.0 with both EE10 and EE11

Also, instead of listing standalone features separately from the main MP features, they are instead included in the main MP repeat actions. This avoids the need to add further custom repeats and makes the EE compatibility tests a little simpler.

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.microprofile.graphql.1.0_fat/fat/src/com/ibm/ws/microprofile/graphql/fat/FATSuite.java [hunk 1]
```diff
@@ -54,19 +54,19 @@ public class FATSuite {
...
+                                                      .withID(JakartaEE9Action.ID+"_mpGraphQL20").conditionalFullFATOnly(FeatureReplacementAction.GREATER_THAN_OR_EQUAL_JAVA_11))
                                              .andWith(new JakartaEE10Action()
                                                       .removeFeatures(setOf("mpConfig-1.4", "mpConfig-2.0")).addFeature("mpConfig-3.0")
                                                       .removeFeatures(setOf("mpMetrics-3.0", "mpMetrics-2.3", "mpMetrics-4.0")).addFeature("mpMetrics-5.0")
                                                       .removeFeatures(setOf("mpRestClient-2.0", "mpRestClient-1.4")).addFeature("mpRestClient-3.0")
                                                       .removeFeature("mpGraphQL-1.0").addFeature("mpGraphQL-2.0").alwaysAddFeature("servlet-6.0")
-                                                      .withID(MicroProfileActions.STANDALONE10_ID));
+                                                      .withID(JakartaEE10Action.ID+"_mpGraphQL20"));
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.graphql.1.0_fat/fat/src/com/ibm/ws/microprofile/graphql/fat/FATSuite.java:69]: This change doesn't look related.

File: dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java [hunk 2]
```diff
@@ -70,208 +67,276 @@ public class MicroProfileActions {
...
                                                           "mpOpenAPI-2.0",
                                                           "mpOpenTracing-2.0",
-                                                          "mpRestClient-2.0" };
+                                                          "mpRestClient-2.0",
+                                                          "mpContextPropagation-1.2", //standalone
+                                                          "mpGraphQL-1.0", //standalone
+                                                          "mpLRA-1.0", //standalone
+                                                          "mpLRACoordinator-1.0", //standalone
+                                                          "mpReactiveMessaging-1.0", //standalone
+                                                          "mpReactiveStreams-1.0" }; //standalone
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java]: `mpReactiveMessaging-1.0` is not compatible with `mpConfig-2.0`. I would remove both reactive features from MP 4.0 and 4.1.

File: dev/com.ibm.ws.microprofile.graphql.1.0_fat/fat/src/com/ibm/ws/microprofile/graphql/fat/FATSuite.java (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.ws.microprofile.graphql.1.0_fat/fat/src/com/ibm/ws/microprofile/graphql/fat/FATSuite.java:69]: It is related because I removed the standalone ID constants ... which this class should not have been using anyway.

**Comment by tevans78** [comment]
#build

**Review by Azquelt — APPROVED** [review]

**Comment by tevans78** [comment]
In one of the 11 full fat runs for mpGraphQL, an intermittent failure was seen...
```
java.lang.IllegalStateException: The method is called on an invalidated session: pxHq7LtBp4XhP9uKr8mxPCz
	at com.ibm.ws.session.http.AbstractHttpSession.isNew(AbstractHttpSession.java:218)
	at com.ibm.ws.webcontainer.session.impl.HttpSessionContextImpl.doSecurityCheck(HttpSessionContextImpl.java:608)
	at com.ibm.ws.webcontainer.session.impl.HttpSessionContextImpl.getIHttpSession(HttpSessionContextImpl.java:551)
	at com.ibm.ws.webcontainer.session.impl.HttpSessionContextImpl.getIHttpSession(HttpSessionContextImpl.java:670)
	at com.ibm.ws.webcontainer.srt.SRTRequestContext.getSession(SRTRequestContext.java:106)
	at com.ibm.ws.webcontainer.srt.SRTServletRequest.getSession(SRTServletRequest.java:2311)
	at com.ibm.ws.webcontainer40.srt.SRTServletRequest40.getSession(SRTServletRequest40.java:284)
	at com.ibm.ws.webcontainer.srt.SRTServletRequest.getSession(SRTServletRequest.java:2294)
	at com.ibm.ws.microprofile.metrics.BaseMetricsRESTProxyServlet.getAndSetRESTHandlerContainer(BaseMetricsRESTProxyServlet.java:90)
	at com.ibm.ws.microprofile.metrics.BaseMetricsRESTProxyServlet.service(BaseMetricsRESTProxyServlet.java:39)
```
Seems extremely unlikely that this was caused by my changing the Repeat ID for that test.

**Comment by tevans78** [comment]
#libby
